ui/src/index.tsx

```diff
@@ -7,7 +7,9 @@ import { Provider as ReduxProvider } from 'react-redux';
 import { store } from './store';
 import { QueryClient, QueryClientProvider } from 'react-query';
 
-const queryClient = new QueryClient();
+const queryClient = new QueryClient({
+  defaultOptions: { queries: { refetchOnWindowFocus: false } },
```

Comment:
Was this causing a problem or we just want to be make less fetches to save on resources?

Reply:
Yeah, this is why it was executing the query every time the window was focused and I think there's no need to do that